tests/testBasisLists.py

```diff
@@ -182,42 +184,42 @@ def testCentralRegularization(self):
         try:
             self.policyDF.set("centralRegularizationStencil", 1)
             ipDiffim.makeRegularizationMatrix(self.policyDF)
-        except Exception, e:  # stencil of 1 not allowed
+        except Exception as e:  # stencil of 1 not allowed
             pass
         else:
             self.fail()
 
         self.policyDF.set("centralRegularizationStencil", 5)
         try:
```

Comment:
I think this entire construct exists to convert an "E" test error into an "F" test failure. @parejkoj would like this but there seems to be far too much boiler plate. I would be inclined to remove all the boiler plate and let any problems trigger an "E" because these commands are all meant to work and an "E" is still something that needs to be fixed. There's a reason why there is no `self.assertDoesNotRaise()` method in `unittest`.